### ISSUE TITLE
Hidden fileds type not hidden after update

### ISSUE THREAD

**Issue by proweb** [htmlburger/carbon-fields#763]
## Version

  - Carbon Fields: 3.1.3
  - WordPress: 5.3
  - PHP: 7.2

## Expected Behavior
Hidden fields NOT visible

## Actual Behavior
Hidden fields render as visible blocks

## Container definition
```php
Container::make( 'post_meta', 'Program choose')
->where( 'post_type', '=', 'kfu_eduprog' )
->set_context( 'carbon_fields_after_title' )
->add_fields( [
	Field::make( 'association', 'kfu_eduprog_association', 'Educational standart' )
	->set_types (array(
		array(
			'type' => 'post', 
			'post_type' => 'obrstandart',
		),
	))
		->set_min( 1 )
		->set_max( 1 )
		->set_required( true ),
	Field::make( 'hidden', 'kfu_eduprog_code_napr', 'code' ),
	Field::make( 'hidden', 'kfu_eduprog_napr', 'Direction' ), // Название направления
	Field::make( 'hidden', 'kfu_eduprog_edu_level', Education level' ),
	Field::make( 'hidden', 'kfu_eduprog_child_id', 'Child' ),


] );
```

## Steps to Reproduce the Problem

  1. Create WordPress installation with carbon fields 2.2 version
  2. Create container with [Field Hidden](https://docs.carbonfields.net/#/fields/hidden)
  3. Save post with data in this hidden fields on post_save action
  4. Update Carbon Fileds from 2.2 to 3 version with conposer or another way
  5. Open post Edit screen with hidden fields.

![Выделение_069](https://user-images.githubusercontent.com/680073/66699877-580d8580-ecf3-11e9-8102-2d58c50055a1.png)

**Comment by piggydoughnut** [comment]
I have exactly the same issue. Has there been any progress on this?

**Comment by elvishp2006** [comment]
+1

**Comment by dhalucario** [comment]
This is still open? Really?

**Comment by dhalucario** [comment]
I created a fix for this. You can now use a `set_hidden(bool)` on the hidden field to hide it.

**Comment by jorostoyanov** [comment]
This is now available in `development` with commit 0a634cb
